developer_docs/ansible_dev_setup.asciidoc

```diff
@@ -67,6 +67,11 @@ Enable feature
 :enabled: true
 ....
 
+Create symlink for `ansible-runner-environment`
+[source, bash]
+....
+sudo ln -s /path/to/gem/smart_proxy_ansible/bin/ansible-runner-environment.sh /usr/libexec/foreman-proxy/ansible-runner-environment
```

Comment:
As was said elsewhere, we should avoid modifying system locations, especially when there are other options.

One would be to do the following before starting smart-proxy
```shell
export SMART_PROXY_ANSIBLE_ENVIRONMENT_WRAPPER="$(bundle exec gem which smart_proxy_ansible | sed 's|/lib/smart_proxy_ansible.rb||')/bin/ansible-runner-environment.sh"
```

One could also use something like direnv to have that done automatically each time they enter a smart proxy directory, but that boils down to personal preference

Reply:
It would probably be worth mentioning that in development, `ansible_environment_file` should be configured in sp-ansible's config and how that file should look like.

Here's an example from one of my machines
```shell
export ANSIBLE_CALLBACK_WHITELIST="theforeman.foreman.foreman"
export ANSIBLE_CALLBACKS_ENABLED="theforeman.foreman.foreman"
export ANSIBLE_LOCAL_TEMP="/tmp"
export ANSIBLE_HOST_KEY_CHECKING="False"
export ANSIBLE_ROLES_PATH="/etc/ansible/roles:/usr/share/ansible/roles"
export ANSIBLE_COLLECTIONS_PATHS="/etc/ansible/collections:/usr/share/ansible/collections"

export FOREMAN_URL="https://april-nightingale.example.com"
export FOREMAN_SSL_CERT="/etc/puppetlabs/puppet/ssl/certs/april-nightingale.example.com.pem"
export FOREMAN_SSL_KEY="/etc/puppetlabs/puppet/ssl/private_keys/april-nightingale.example.com.pem"
export FOREMAN_SSL_VERIFY="/etc/puppetlabs/puppet/ssl/certs/ca.pem"

export ANSIBLE_SSH_ARGS="-o ProxyCommand=none -C -o ControlMaster=auto -o ControlPersist=60s"
```

Reply:
Does this mean that if users set the `ansible_environment_file` then the configurations in `/etc/ansible/ansible.cfg` or `~/.ansible.cfg` are going to be ignored?